proposals/csharp-9.0/module-initializers.md

```diff
@@ -2,7 +2,7 @@
 
 * [x] Proposed
 * [ ] Prototype: [In Progress](https://github.com/jnm2/roslyn/tree/module_initializer)
-* [ ] Implementation: [In Progress](https://github.com/dotnet/roslyn/tree/features/module-initializers)
+* [ ] Implementation: In Progress
```

Comment:
https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md

This document states that this feature implementation is "Merged in 16.7p4".